packages/core/src/tools/internal/inline-tools/link/index.ts

```diff
@@ -87,11 +87,6 @@ export default class LinkInlineTool implements InlineTool {
 
     linkInput.addEventListener('keydown', (event: KeyboardEvent) => {
       if (event.key === 'Enter') {
-        /**
-         * Remove link input, when data formed and trigger callback
-         */
-        linkInput.remove();
-
```

Comment:
?why this change is needed?

Reply:
Just to keep link input there, doesn't matter atm as UX will be updated anyway